src/Shipment/Request/PostReturnShipmentsRequest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Shipment\Request;
+
+use MyParcelNL\Pdk\Account\Request\AbstractRequest;
+use MyParcelNL\Pdk\Base\Support\Collection;
+use MyParcelNL\Pdk\Shipment\Collection\ShipmentCollection;
+use MyParcelNL\Pdk\Shipment\Model\DeliveryOptions;
+use MyParcelNL\Pdk\Shipment\Model\Shipment;
+
+class PostReturnShipmentsRequest extends AbstractRequest
+{
+    /**
+     * @var string
+     */
+    protected $path = '/shipments';
+
+    /**
+     * @var \MyParcelNL\Pdk\Shipment\Collection\ShipmentCollection
+     */
+    private $collection;
+
+    /**
+     * @var array
+     */
+    private $parameters;
+
+    /**
+     * @param  \MyParcelNL\Pdk\Shipment\Collection\ShipmentCollection $collection
+     */
+    public function __construct(ShipmentCollection $collection, array $parameters)
+    {
+        $this->collection = new Collection($collection->all());
+        $this->parameters = $parameters;
+    }
+
+    /**
+     * @return null|string
+     * @throws \MyParcelNL\Pdk\Base\Model\InvalidCastException
+     */
+    public function getBody(): string
+    {
+        return json_encode([
+            'data' => [
+                'return_shipments' => $this->encodeReturnShipments(),
+            ],
+        ]);
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getHeaders(): array
+    {
+        return [
+            'Content-Type' => 'application/vnd.return_shipment+json; charset=utf-8',
+        ];
+    }
+
+    /**
+     * @return string
+     */
+    public function getHttpMethod(): string
+    {
+        return 'POST';
+    }
+
+    /**
+     * @return array
+     */
+    protected function getQueryParameters(): array
+    {
+        return array_filter($this->parameters);
+    }
+
+    /**
+     * @param  \MyParcelNL\Pdk\Shipment\Model\Shipment $shipment
+     *
+     * @return null|array
+     * @throws \MyParcelNL\Pdk\Base\Model\InvalidCastException
+     */
+    private function encodeReturnOptions(Shipment $shipment): ?array
+    {
+        $shipmentOptions = $shipment->deliveryOptions->shipmentOptions;
+
+        if ($shipmentOptions) {
```

Comment:
Hoe kan $shipmentOptions falsey zijn? Komt er niet altijd een ShipmentOptions uit de DeliveryOptions, ook al is die misschien leeg? - just checking